chromium_src/chrome/browser/media/router/media_router_feature.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/media/router/media_router_feature.h"
+
+#include "base/feature_list.h"
+#include "build/build_config.h"
+
+#define MediaRouterEnabled MediaRouterEnabled_ChromiumImpl
+#include "../../../../../../chrome/browser/media/router/media_router_feature.cc"
+#undef MediaRouterEnabled
+
+namespace media_router {
+
+bool MediaRouterEnabled(content::BrowserContext* context) {
+#if !defined(OS_ANDROID)
+  if (!base::FeatureList::GetInstance()->IsFeatureOverridden(
```

Comment:
Nit: Maybe a small comment to clarify that this is done to override the default could be useful.